packages/mobile/src/store/sagas.ts

```diff
@@ -103,6 +104,9 @@ export default function* rootSaga() {
     // Tipping
     ...tippingSagas(),
 
+    // Premium content
+    ...premiumContentSagas(),
```

Comment:
How come we have premiumContentSagas in both mobile + common sagas?

Reply:
not sure wym. followed the chatSagas pattern here. wasn't aware that there is redundancy?